sdk/storage/storage-queue/tests.yml

```diff
@@ -12,3 +12,8 @@ jobs:
   - template: ../archetype-sdk-tests-storage.yml
     parameters:
       PackageName: "@azure/storage-queue"
+      EnvVars:
+        ACCOUNT_NAME: $(js-storage-test-account-name)
```

Comment:
These EnvVars override the ones provided at `../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml`.
Overriding would mean that storage-queue is still relying on the manually created storage account(resource) rather than depending on the template.